README.md

```diff
@@ -14,6 +14,15 @@ tweetnacl.h -> 02aee579efbe4f12ad8ffb23c6772841
 ## Fixes
 Change integer size in commit [b98e0ca](https://github.com/RIOT-OS/tweetnacl/commit/b98e0ca8beb9b802b5c0492d88846bd9de9946eb)
 
+Fix a warning on line 58 about comparison between two different integer
+sizes.
```

Comment:
`int` is 32-bit on (most) 32- and 64-bit platforms while being 16-bit on (most) 8- and 16-bit platforms, so arguing about fixing a warning about size is just false. If anything you are *introducing* an error about different sizes with b98e0ca8beb9b802b5c0492d88846bd9de9946eb.